app/src/main/java/pl/gittobefit/user/fragments/Logout.java

```diff
@@ -59,12 +59,12 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                 mGoogleApiClient.signOut();
```

Comment:
Jeżeli jest to zmienna to:
```suggestion
                googleApiClient.signOut();
```